views/layout.hbs

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <link rel="stylesheet" href="/css/style.css">
+      <link rel="stylesheet" href="/css/responsive.css">
+      <link rel="stylesheet" href="/css/flexboxgrid.css">
+      <link rel="stylesheet" href="/css/jquery.bxslider.css">
+      <link href="https://fonts.googleapis.com/css?family=Dosis:400,500,600,700" rel="stylesheet">
+      <title>Training Exercise</title>
+  </head>
+  <body>
+    {{>header}}
+    {{{body}}}
+    {{>footer}}
+    <script src="/js/handlebars-v4.1.1.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.0/jquery.js"></script>
```

Comment:
Din't you include jquery in bundle.js above ?

Reply:
Removed bundle.js file.